routing/index_graph_starter_joints.cpp

```diff
@@ -213,9 +215,11 @@ void IndexGraphStarterJoints::GetEdgeList(JointSegment const & vertex, bool isOu
     {
       size_t prevSize = edges.size();
       AddFakeJoints(jointEdges[i].GetTarget().GetSegment(!opposite /* start */), isOutgoing, edges);
-      // Если мы добавили фейковое ребро, то надо сохранить вес "ребенок[i]" -> "родитель"
-      // Так как у фекового ребра стартовый сегмент это i-ый ребенок, то мы и сохраняем
-      // тот же самый вес |parentWeight[i]|.
+
+      // If we add fake edge, we should add new parentWeight as "child[i] -> parent".
+      // Because fake edge and current edge - jointEdges[i], has the same first
```

Comment:
because fake edge and current edge are jointEdges[i] and have the same first segments

Reply:
done

Reply:
```
// If we add fake edge, we should add new parentWeight as "child[i] -> parent".
      // Because fake edge and current edge (jointEdges[i]) have the same first
      // segments (differ only the ends), so we add to |parentWeights| the same
      // value: parentWeights[i].
```